template/template.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="template.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+      <link rel="icon" type="image/x-icon" href="speed_logo-cropped.png">
+</head>
+<body>
+    <header>
+        <div id="logo-container">
+        <img id="logo" src="speed logo.svg" alt="logo">
+        <div id="logo-name">Speed CL</div>
+        </div>
+        <div id="header-icon-container">
+        <img src="dark-mode.svg" alt="dark mode icon" id="dark-mode">
+        <img src="github.svg" alt="github icon" id="github-icon">
+        </div>
+    </header>
+    <div id="main-content">
+        <div id="component">
+        </div>
+        <aside id="side-bar">
+            <ul id="side-bar-links">
+                <li class="side-bar-item"><a href="#">Avatar</a></li>
+                <li class="side-bar-item"><a href="#">Alert</a></li>
+                <li class="side-bar-item"><a href="#">Badge</a></li>
+                <li class="side-bar-item"><a href="#">Butto</a></li>
+                <li class="side-bar-item"><a href="#">Card</a></li>
+                <li class="side-bar-item"><a href="#">Image</a></li>
+                <li class="side-bar-item"><a href="#">Input</a></li>
+                <li class="side-bar-item"><a href="#">Text Utilities</a></li>
+                <li class="side-bar-item"><a href="#">Lists</a></li>
+                <li class="side-bar-item"><a href="#">Navigation</a></li>
+                <li class="side-bar-item"><a href="#">Modal</a></li>
+                <li class="side-bar-item"><a href="#">Rating</a></li>
+            </ul>
+          </aside>
+    </div>
+    <script src="app.js"></script>
+</body>
+</html>
```

Comment:
You can enable 'add new line at the end' feature in vs code to get rid of this error.
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files